### PR TITLE
C411857 Match profile: update options for Item "Incoming records" (folijet) (TaaS)

### PR DESCRIPTION
**Pull request by KirylShup**
[Allure Report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/6135/allure/#)
[Test case 411857](https://foliotest.testrail.io/index.php?/cases/view/411857)
image

### CODE REVIEW

File: cypress/e2e/data-import/settings/update-options-for-item-incoming-records-in-match-profile.cy.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      });
+    });
+
+    it(
+      'C411857 Match profile: update options for Item "Incoming records" (folijet) (TaaS)',
+      { tags: ['extendedPath', 'folijet'] },
+      () => {
+        cy.visit(SettingsMenu.matchProfilePath);
+        MatchProfiles.openNewMatchProfileForm();
+        NewMatchProfile.selectExistingRecordType('ITEM');
```
Comment (manowarqqq) [inline cypress/e2e/data-import/settings/update-options-for-item-incoming-records-in-match-profile.cy.js:30]: use EXISTING_RECORDS_NAMES.ITEM constant

File: cypress/support/fragments/data_import/match_profiles/newMatchProfile.js [hunk 2]
```diff
@@ -564,6 +564,12 @@ export default {
       DropdownMenu({ visible: true }).find(HTML('MARC Bibliographic')).exists(),
       DropdownMenu({ visible: true }).find(HTML('Static value (submatch only)')).exists(),
     ]);
+    cy.do(Dropdown({ id: 'record-selector-dropdown' }).toggle());
```
Comment (manowarqqq) [inline cypress/support/fragments/data_import/match_profiles/newMatchProfile.js:567]: please, verify that it doesn't break existing tests that use this method

File: cypress/support/fragments/data_import/match_profiles/newMatchProfile.js [hunk 3]
```diff
@@ -564,6 +564,12 @@ export default {
...
+  },
+  verifyAbsenceInIncomingRecordsDropdown: (option) => {
+    cy.do(Dropdown({ id: 'record-selector-dropdown' }).toggle());
```
Comment (manowarqqq) [inline cypress/support/fragments/data_import/match_profiles/newMatchProfile.js:570]: move to variable

File: cypress/support/fragments/data_import/match_profiles/newMatchProfile.js [hunk 4]
```diff
@@ -564,6 +564,12 @@ export default {
...
+    cy.expect(DropdownMenu({ visible: true }).find(HTML(option)).absent());
+    cy.do(Dropdown({ id: 'record-selector-dropdown' }).toggle());
```
Comment (manowarqqq) [inline cypress/support/fragments/data_import/match_profiles/newMatchProfile.js:572]: find(Button(option)...

**Review by manowarqqq — APPROVED** [review]

**Review by TetianaParanich — APPROVED** [review]

**Pull request merged**